osu.Game/osu.Game.csproj

```diff
@@ -34,6 +34,11 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
+    <PackageReference Include="ppy.ManagedBass" Version="2023.0.1" />
+    <PackageReference Include="ppy.ManagedBass.Enc" Version="3.0.0" />
```

Comment:
What is `ppy.ManagedBass.Enc`? It doesn't look to exist.

Reply:
Read through the PR prerequisites. BassEnc is a dependency of BassLoud

Reply:
I've since removed bassenc